deps/hiredis/sds.c

```diff
@@ -1020,7 +1020,13 @@ sds *sdssplitargs(const char *line, int *argc) {
                 if (*p) p++;
             }
             /* add the token to the vector */
-            vector = s_realloc(vector,((*argc)+1)*sizeof(char*));
+            char **data = s_realloc(vector,((*argc)+1)*sizeof(char*));
+            if (data == NULL) {
+            	return NULL;
```

Comment:
I'm not sure if returning null is what we want to do here. Please let me know if this should change.